raiden/api/python.py

```diff
@@ -1643,25 +1643,29 @@ def register_light_client(self,
 
             api_key = hexlify(os.urandom(20))
             api_key = api_key.decode("utf-8")
-
-            result = self.raiden.wal.storage.save_light_client(
-                api_key=api_key,
-                address=address,
-                encrypt_signed_password=encrypt_signed_password.hex(),
-                encrypt_signed_display_name=encrypt_signed_display_name.hex(),
-                encrypt_signed_seed_retry=encrypt_signed_seed_retry.hex())
-
-            if result > 0:
-                result = {"address": address,
-                          "encrypt_signed_password": encrypt_signed_password.hex(),
-                          "encrypt_signed_display_name": encrypt_signed_display_name.hex(),
-                          "api_key": api_key,
-                          "encrypt_signed_seed_retry": encrypt_signed_seed_retry.hex(),
-                          "message": "successfully registered",
-                          "result_code": 0}
+            #Check for limit light client
+            if HUB_MAX_LIGHT_CLIENTS > len(self.get_all_light_clients()):
+                result = self.raiden.wal.storage.save_light_client(
+                    api_key=api_key,
+                    address=address,
+                    encrypt_signed_password=encrypt_signed_password.hex(),
+                    encrypt_signed_display_name=encrypt_signed_display_name.hex(),
+                    encrypt_signed_seed_retry=encrypt_signed_seed_retry.hex())
+
+                if result > 0:
+                    result = {"address": address,
+                              "encrypt_signed_password": encrypt_signed_password.hex(),
+                              "encrypt_signed_display_name": encrypt_signed_display_name.hex(),
+                              "api_key": api_key,
+                              "encrypt_signed_seed_retry": encrypt_signed_seed_retry.hex(),
+                              "message": "successfully registered",
+                              "result_code": 0}
+                else:
+                    result = {"message": "An unexpected error has occurred.",
+                              "result_code": 1}
             else:
-                result = {"message": "An unexpected error has occurred.",
-                          "result_code": 1}
+                result = {"message": "Limit of registered light clients reached.",
+                          "result_code": 3}
```

Comment:
There is no HTTP status code for a quota or limit exceeded, but i think 403 forbidden is ok with that message.